scripts/orion

```diff
@@ -0,0 +1,46 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Name 'orion' for this file doesn't provide hints about in which way this script is different from just calling contextBroker...

Maybe 'contextBrokerFg' would be a better name?